test/extended/networking/network_segmentation.go

```diff
@@ -1474,7 +1474,10 @@ func assertClusterUDNStatusReportsActiveNamespaces(cudnName string, expectedActi
 	// equality matcher cannot be used since condition message namespaces order is inconsistent
 	ExpectWithOffset(1, c.Type).Should(SatisfyAny(Equal("NetworkReady"), Equal("NetworkCreated")))
 	ExpectWithOffset(1, c.Status).Should(Equal(metav1.ConditionTrue))
-	ExpectWithOffset(1, c.Reason).Should(Equal("NetworkAttachmentDefinitionReady"))
+	ExpectWithOffset(1, c.Reason).Should(Or(
+		Equal("NetworkAttachmentDefinitionReady"),
```

Comment:
Would be good to create a followup task to clean it up later on.

Reply:
Like spoken yesterday @trozet we should break this out into a new PR and get this in..
the more the jobs go read the longer it takes to get the percentile up, so the faster we get this fix the better we are..

@kyrtapz can you please break this out and talk with @trozet when is online for lgtm/approve -> AFAICT this is a downstream only problem....upstream its all OK.

for the original comment, yes we @ormergi has to do a followup

Reply:
https://github.com/openshift/origin/pull/29446